### DISCUSSION TITLE
Voti

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#577] [Richieste di aiuto] [answered]
Salve io vorrei mettere i voti dal 6- al 5 non in giallo ma in rosso coke potrei fare?

**Comment by trinko** [comment, marked as answer]
Attualmente l'unico modo è modificare i template dove vengono visualizzati.
Esempio per la pagina del quadro voti:
- copia il file **templates/lezioni/voti_quadro.html.twig** in  **PERSONAL/templates/lezioni/voti_quadro.html.twig**
- nel nuovo file, modifica la riga 131 nel modo seguente:
```
          <button type="button" class="btn btn-xs btn-{{ v.voto < 6 ? 'danger' :  'success' }}" title="Informazioni sulla valutazione" data-toggle="modal" data-target="#gs-modal-info" data-title="Informazioni sulla valutazione" data-body="gs-voti-{{ v.id }}">{{ v.voto_str }}</button>
```
Dopo aver salvato il file, devi svuotare la cache (come amministratore, **SISTEMA > MANUTENZIONE**)

**Comment by Davide031** [comment]
Quando io svuoto la chache in pratica carica ma poi non esce chache svuotata con successo e non cambia nulla

**Reply by trinko** [reply]
Un altro modo per svuotare la cache: cancella tutto il contenuto della cartella **var/cache/prod/**

**Comment by Davide031** [comment]
Fatto ma i voti risultano gialli

**Comment by trinko** [comment]
Stai guardando la pagina del quadro voti del docente?

**Comment by Davide031** [comment]
Si ma risulta giallo
